consensus/core/src/block_verifier.rs

```diff
@@ -158,9 +158,7 @@ impl BlockVerifier for SignedBlockVerifier {
         }
 
         let max_num_transactions_limit =
-            self.context
-                .protocol_config
-                .consensus_max_num_transactions_in_block() as usize;
+            self.context.protocol_config.max_num_transactions_in_block() as usize;
```

Comment:
nit: Since you are renaming this maybe let's also do the same for `consensus_max_transactions_in_block_bytes` to `max_transactions_in_block_bytes` to align them? Or vice versa

Reply:
The `max_num_transactions_in_block()` wrapper is added to provide a default for older protocol configs. I will remove that in future after live networks have upgraded.